models/gc_picai_baseline/meta.json

```diff
@@ -0,0 +1,179 @@
+{
+  "id": "c5f886fb-9f54-4555-a954-da02b22d6d3f",
+  "name": "picai_baseline",
```

Comment:
Just noticed, this should be `gc_picai_baseline` ;)

Reply:
Done.